resources/rtos.css

```diff
@@ -20,18 +20,23 @@
   padding: 0px;
   font-family: var(--vscode-editor-font-family);
   font-weight: var(--vscode-editor-font-weight);
-  font-size: var(--vscode-editor-font-size); 
+  font-size: var(--vscode-editor-font-size);
 }
 
 vscode-link {
   font-family: var(--vscode-editor-font-family);
   font-weight: var(--vscode-editor-font-weight);
-  font-size: var(--vscode-editor-font-size);  
+  font-size: var(--vscode-editor-font-size);
 }
 
 .running {
   color: var(--vscode-symbolIcon-classForeground);
-  font-weight:bold;
+  font-weight: bold;
+}
+
+.backgroundPercent {
+  background: linear-gradient(to right, var(--vscode-charts-blue) 0% var(--rtosview-percentage-active, 0%), rgba(0, 0, 0, 0) var(--rtosview-percentage-active, 0%) 100%);
+  background-clip: padding-box;
 }
 
```

Comment:
Nice! I have to see how this works after I merge

Reply:
Jup, is a quite nice solution. Also experimented with mix-blend-mode to get the text inverted when inside the active part, but didn't got it working